modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/HeadlessDataSetDisplayTag.java

```diff
@@ -72,7 +79,11 @@ public int doStartTag() throws JspException {
 		return super.doStartTag();
 	}
 
-	public String getAPIURL() {
+	public String getActionParameterName() {
+		return _actionParameterName;
+	}
+
+	public String getApiURL() {
```

Comment:
So the rule is "acronyms get capitalized, unless they are consecutive, in which case only the last one does"?

![doesnt-make-sense](https://user-images.githubusercontent.com/7074/90383154-0a367c00-e080-11ea-9796-fb7d727692cd.gif)

Reply:
That's actually an exception to the rule and I'm not sure how Brian will feel about it. The thing is that a TagLib has its attributes defined on a `.tld` file, right. And in there we define `apiURL` as an attribute. That implies that we need capitalized getters and setters in the Java class, but when I had `getAPIURL` it was throwing an error saying it couldn't find them. So that's the reason for this change. Not sure there's a way around it.